sys/uuid/uuid.c

```diff
@@ -110,3 +111,20 @@ void uuid_v5(uuid_t *uuid, const uuid_t *ns, const uint8_t *name, size_t len)
     _set_version(uuid, UUID_V5);
     _set_reserved(uuid);
 }
+
+void uuid_to_string(char *str, const uuid_t *uuid)
+{
+    char *p = str;
+    p += fmt_u32_hex(p, byteorder_ntohl(uuid->time_low));
+    p += fmt_str(p, "-");
```

Comment:
Minor Nit: using `*p++ = '-';` instead of the function call here and below saves 80 bytes in the unittest.